app/core/enterprise_auth.py

```
@@ -0,0 +1,632 @@
+            # Remove from Redis
+            if self.redis_client:
+                await self.redis_client.delete(f"session:{session_id}")
+            
+            # Deactivate database session
+            try:
+                await enterprise_db.update_record(
+                    UserSession,
+                    session_id,
+                    {"is_active": False, "logged_out_at": datetime.now(timezone.utc)}
+                )
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)